framework/transport/ieee1905_transport/ieee1905_transport_netlink.cpp

```diff
@@ -117,4 +118,5 @@ void Ieee1905Transport::handle_netlink_pollin_event()
     return;
 }
 
-} // namespace mapf
\ No newline at end of file
+} // namespace transport
+} // namespace beerocks
```

Comment:
missing newline at the end of file

Reply:
It's there, simply not displayed in the diff.
GitHub displays an icon when there's no new line at the end of a file.